### ISSUE TITLE
This code is now deprecated, is there an ETA when this API will no longer work?

### ISSUE THREAD

**Issue by danzada1** [PhilipsHue/PhilipsHueSDK-Java-MultiPlatform-Android#46]
This code is now deprecated, is there an ETA when this API will no longer work?

Thanks!

**Comment by TobseF** [comment]
Yes you are right, The [Philips HUE SDK was **deprecated** on July 1st 2019](https://developers.meethue.com/deprecation-philips-hue-sdk/). So it won't get any updates anymore. 
> Existing Hue SDK client applications will continue to work normally but we can no longer guarantee long term compatibility with future Hue software.

They favor the REST API of the bridge. The offical [Get Started guide](https://developers.meethue.com/develop/get-started-2/) shows the easy usage of this API.

And if if you want a simple modern typesafe Java API which you can include as Maven / Gradle dependency try the : :point_right:[**yetanotherhueapi**](https://zeroone3010.github.io/yetanotherhueapi/).

So there is no need anymore to include this **PhilipsHueSDK** jars in your current projects.
